### PR TITLE
Add httparty and multi_xml

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
These will be needed when SCLing the smart proxy packages, I wanted to get them in ahead of time.

File: packages/plugins/rubygem-httparty/rubygem-httparty.spec [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Generated from httparty-0.17.0.gem by gem2rpm -*- rpm-spec -*-
```
Comment (evgeni) [inline packages/plugins/rubygem-httparty/rubygem-httparty.spec]: I think we dropped that from the template (no harm tho)

File: packages/plugins/rubygem-httparty/rubygem-httparty.spec [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
```
Comment (evgeni) [inline packages/plugins/rubygem-httparty/rubygem-httparty.spec]: old template is old ;)

File: packages/plugins/rubygem-httparty/rubygem-httparty.spec [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+mkdir -p %{buildroot}%{_bindir}
+cp -pa .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+
+%files
+%dir %{gem_instdir}
+%{_bindir}/httparty
+%{gem_instdir}/.editorconfig
```
Comment (evgeni) [inline packages/plugins/rubygem-httparty/rubygem-httparty.spec]: exclude

File: packages/plugins/rubygem-httparty/rubygem-httparty.spec [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.rubocop.yml
+%exclude %{gem_instdir}/.rubocop_todo.yml
+%{gem_instdir}/.simplecov
```
Comment (evgeni) [inline packages/plugins/rubygem-httparty/rubygem-httparty.spec]: exclude

File: packages/plugins/rubygem-httparty/rubygem-httparty.spec [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+%exclude %{gem_instdir}/.travis.yml
+%{gem_instdir}/Guardfile
```
Comment (evgeni) [inline packages/plugins/rubygem-httparty/rubygem-httparty.spec]: exclude

File: packages/plugins/rubygem-httparty/rubygem-httparty.spec [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+%license %{gem_instdir}/MIT-LICENSE
+%{gem_instdir}/bin
+%{gem_libdir}
+%{gem_instdir}/script
+%{gem_instdir}/website
```
Comment (evgeni) [inline packages/plugins/rubygem-httparty/rubygem-httparty.spec]: exclude

File: packages/plugins/rubygem-httparty/rubygem-httparty.spec [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.rubocop.yml
+%exclude %{gem_instdir}/.rubocop_todo.yml
+%{gem_instdir}/.simplecov
+%exclude %{gem_instdir}/.travis.yml
+%{gem_instdir}/Guardfile
+%license %{gem_instdir}/MIT-LICENSE
+%{gem_instdir}/bin
+%{gem_libdir}
+%{gem_instdir}/script
```
Comment (evgeni) [inline packages/plugins/rubygem-httparty/rubygem-httparty.spec]: exclude

File: packages/plugins/rubygem-httparty/rubygem-httparty.spec (hunk 6 above)
Comment (evgeni) [reply packages/plugins/rubygem-httparty/rubygem-httparty.spec]: or maybe move to docs?

**Comment by ehelms** [comment]
Updated - I swear I had reset my branch

File: packages/plugins/rubygem-httparty/rubygem-httparty.spec [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+%{gem_spec}
+
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/CONTRIBUTING.md
+%doc %{gem_instdir}/Changelog.md
+%{gem_instdir}/Gemfile
+%doc %{gem_instdir}/README.md
+%{gem_instdir}/Rakefile
+%{gem_instdir}/cucumber.yml
```
Comment (evgeni) [inline packages/plugins/rubygem-httparty/rubygem-httparty.spec]: exclude?

File: packages/plugins/rubygem-multi_xml/rubygem-multi_xml.spec [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+Requires: %{?scl_prefix_ruby}ruby(rubygems) >= 1.3.5
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel >= 1.3.5
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+Provides swappable XML backends utilizing LibXML, Nokogiri, Ox, or REXML.
```
Comment (evgeni) [inline packages/plugins/rubygem-multi_xml/rubygem-multi_xml.spec]: should any of these be in `Requires`?

Comment (ehelms) [reply packages/plugins/rubygem-multi_xml/rubygem-multi_xml.spec]: I checked the EPEL version and it had no requires on any of those.

File: comps/comps-foreman-plugins-rhel7.xml [hunk 10]
```diff
@@ -133,6 +133,7 @@
       <packagereq type="default">tfm-rubygem-jgrep</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-plugins-rhel7.xml]: shouldn't httpparty (non doc) be here somewhere too?

**Pull request merged**